tsconfig.test.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["lib/**/*.ts"]
-}
+  "include": ["lib/**/*.ts", "test/**/*.ts"]
```

Comment:
This is a drive-by fix - the tests are quite neat but they weren't actually running for me. I believe this was the intent?

Reply:
Good spot, I broke it a couple of commits ago! Thanks for fixing it!